salt/iiif/config/usr-local-bin-loris-cache-clean

```diff
@@ -36,16 +36,17 @@ if [ "$usage" -lt "$soft_limit" ]; then
     exit 0
 fi
 
-systemctl stop nginx
-
 if [ "$usage" -lt "$hard_limit" ]; then
-    log_message "Starting loris-cache-clean-soft"
-    timeout 240 /usr/local/bin/loris-cache-clean-soft "$soft_limit" || log_message "TIMEOUT OF loris-cache-clean-soft"
-    log_message "Completed loris-cache-clean-soft"
+#    systemctl stop nginx
+#    log_message "Starting loris-cache-clean-soft"
+#    timeout 240 /usr/local/bin/loris-cache-clean-soft "$soft_limit" || log_message "TIMEOUT OF loris-cache-clean-soft"
+#    log_message "Completed loris-cache-clean-soft"
+#    systemctl start nginx
+    log_message "Skipping loris-cache-clean-soft (disabled)"
 else
+    systemctl stop nginx
```

Comment:
correct for me, will only get here if usage is greater than the hard limit

Reply:
for reference, the limits are quite similar: https://github.com/elifesciences/builder-private/blob/4586ba5a382476829d28390ab62caefcb8ba3df7/pillar/environment-prod.sls#L115-L116